src/app/core/services/notification.service.ts

```diff
@@ -345,8 +345,9 @@ export class NotificationService {
   cancelNotifications() {
     return this.storage
       .get(StorageKeys.PARTICIPANTLOGIN)
-      .then(participantLogin => {
-        return this.cancelNotificationPush(participantLogin)
-      })
+      .then(
+        participantLogin =>
+          participantLogin && this.cancelNotificationPush(participantLogin)
```

Comment:
It should check the same when scheduling the notification otherwise it may result in an inconsistent state

Reply:
Made the changes.